admin-dev/filemanager/execute.php

```diff
@@ -34,12 +34,7 @@
 
 $base = $current_path;
 
-if (isset($_POST['path'])) {
-    $path = $current_path.str_replace("\0", '', $_POST['path']);
-} else {
-    $path = $current_path;
-}
-
+$path = isset($_POST['path']) ? ($current_path . $_POST['path']) : $current_path;
```

Comment:
why change the original code? this str_replace is is used for legitimate safety reasons (null byte injection)
we recently fixed a few security breaches in this component, so I prefer not to modify its behavior if not necessary